api/src/main/java/org/pmiops/workbench/db/model/DbDemographicSurvey.java

```diff
@@ -174,6 +181,34 @@ public void setRaceEnum(List<Race> raceList) {
         raceList.stream().map(DemographicSurveyEnum::raceToStorage).collect(Collectors.toList());
   }
 
+  @ElementCollection(fetch = FetchType.LAZY)
+  @CollectionTable(
+      name = "demographic_survey_gender_identity",
+      joinColumns = @JoinColumn(name = "demographic_survey_id"))
+  @Column(name = "gender_identity")
+  public List<Short> getGenderIdentityList() {
+    return genderIdentityList;
+  }
+
+  public void setGenderIdentityList(List<Short> genderIdentityList) {
```

Comment:
nit: can this be a set instead of a list?

Reply:
Good point will change this to set

Reply:
However this will . be consistent with other elements will it be ok if i can have another PR to have it for all the elements (race ,sexAtbirth)?